### ISSUE TITLE
Incorrect error bands on familiar/ novel looking time plot

### ISSUE THREAD

**Issue by mzettersten** [gongcastro/Flip#6]
@gongcastro I *think* that the error bands in the familiar/ novel looking time figure (03_interaction) are incorrect, in the sense that they do not reflect estimates from the lmer model.

As far as I can tell, the error bands are generated with geom_smooth in line 325 in flip.R:

`geom_smooth(method = "lm", formula = "y ~ x", alpha = 0.25)`

The result of the error bands is therefore from a linear model fit of LookingTime ~ HPP, collapsing across all of the data points without respecting e.g. non-independence due to subjects. I suspect this is why the error bands look odd.

Basically, we want to plot the error bands based on our actual model estimates.

I'm currently working on some possible options - as well as some alternate figure representations.

**Comment by mzettersten** [comment]
Note that I think the prediction lines are actually also not from the model's predictions, for a similar reason as the error bands.

**Comment by mzettersten** [comment]
One way I have learned to plot the model predictions from an lmer model in the past is to use the predictSE function from the library AICcmodavg [https://www.rdocumentation.org/packages/AICcmodavg/versions/2.2-2/topics/predictSE](url)

I'm not sure that's really the best option, but it's not terrible (and should be pretty close to the output of what the "ideal" solution might look like, i.e. computing the standard errors using the same procedure as they are estimated in the model, e.g. bootstrapping, I guess?).

**Comment by gongcastro** [comment]
@mzettersten I have addressed this issue in the last commit. Model predictions and SE bands are now generated using `predictSE()` from the AICcmodavg package, as you suggested. I have integrated you code from flip_plot_temp.R into the main script. AFAICT, the resulting plots are identical to yours, so they *should* be correct!

**Comment by mzettersten** [comment]
sweet! Thanks for all your work on that!

I'm going to put into my mental notebook that I want to go back and develop a better workflow for  plotting SEs/ CIs from lmer and glmer models - but not important for this particular project. Will share if I settle on something I like better (and let me know if you find something you really like!).
